src/overlays/actors/ovl_Demo_Im/z_demo_im.c

```diff
@@ -107,7 +107,7 @@ extern AnimationHeader D_06000AFC;
 extern Gfx D_0600EDE8[];
 extern AnimationHeader D_060014E4;
 extern AnimationHeader D_06001868;
-extern FlexSkeletonHeader D_0600F788;
+extern FlexSkeletonHeader object_im_Skel_00F788;
```

Comment:
Shouldn't you be removing this and adding the header?
Also, what about the other symbols?